graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -787,9 +786,6 @@ def _compile_root_ast_to_ir(schema, ast, type_equivalence_hints=None):
         # 'type_equivalence_hints_inverse' is the inverse of type_equivalence_hints,
         # which is always invertible.
         'type_equivalence_hints_inverse': invert_dict(type_equivalence_hints),
-        # 'subclass_sets' is a dict mapping class names to the set of its subclass names
-        'subclass_sets': compute_subclass_sets(
```

Comment:
There is no usage of 'subclass_sets' field in the entire codebase so I felt free to delete it. @bojanserafimov is there an use case I am missing?

Reply:
I think the use might not have gotten added yet, since this is an unmerged branch :)